src/Parsing/Impl/Parser.cs

```diff
@@ -2025,66 +2025,66 @@ struct Name {
                         continue;
                     } else if (!allowAnnotations) {
-                        ReportSyntaxError(p.StartIndex, p.EndIndex, "invalid syntax");
+                        ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.InvalidSyntaxErrorMsg);//invalid syntax
                         continue;
                     }
                 }
 
                 if (p.DefaultValue == null) {
                     if (seenDefault && p.Kind == ParameterKind.Normal) {
-                        ReportSyntaxError(p.StartIndex, p.EndIndex, "default value must be specified here");
+                        ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.DefaultValueMustBeSpecifiedErrorMsg);//default value must be specified here
                     }
                 } else {
                     seenDefault = true;
                 }
 
                 if (p is SublistParameter sp) {
                     if (_stubFile || _langVersion.Is3x()) {
-                        ReportSyntaxError(p.StartIndex, p.EndIndex, "sublist parameters are not supported in 3.x");
+                        ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.SublistParametersNotSupported3dotXErrorMsg);//sublist parameters are not supported in 3.x
                     } else {
                         ValidateSublistParameter(sp.Tuple?.Items, seenNames);
                     }
                     continue;
                 }
 
                 if (p is ErrorParameter) {
-                    ReportSyntaxError(p.StartIndex, p.EndIndex, "invalid parameter");
+                    ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.InvalidParameterErrorMsg);//Invalid Parameter
                     continue;
                 }
 
                 if (string.IsNullOrEmpty(p.Name)) {
                     if (p.Kind != ParameterKind.List || !(_stubFile || _langVersion.Is3x())) {
-                        ReportSyntaxError(p.StartIndex, p.EndIndex, "invalid syntax");
+                        ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.InvalidSyntaxErrorMsg);//Invalid Syntax
                         continue;
                     }
                 } else if (!seenNames.Add(p.Name)) {
-                    ReportSyntaxError(p.StartIndex, p.EndIndex, "duplicate argument '{0}' in function definition".FormatUI(p.Name));
+                    ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.DuplicateArgumentInFunctionErrorMsg.FormatUI(p.Name));
                 }
 
                 if (p.Kind == ParameterKind.List) {
                     if (seenListArg) {
-                        ReportSyntaxError(p.StartIndex, p.EndIndex, "duplicate * args arguments");
+                        ReportSyntaxError(p.StartIndex, p.EndIndex, Resources.DuplicateArgsArgumentErrorMsg.FormatInvariant("*"));//duplicate * args arguments
```

Comment:
This probably does not need formatting, `*` can be part of the message as before.

Reply:
Made recommended changes.